web/src/components/__tests__/__snapshots__/Layout.spec.tsx.snap

```diff
@@ -9,6 +9,7 @@ exports[`Layout should render all components 1`] = `
       <header />
```

Comment:
Ughhh ugly snapshot :face_with_thermometer: We should just remove those, nobody is looking at them anyway...